paimon-core/src/main/java/org/apache/paimon/operation/PartitionExpire.java

```diff
@@ -117,6 +124,10 @@ private List<BinaryRow> readPartitions(LocalDateTime expireDateTime) {
                 .collect(Collectors.toList());
     }
 
+    public void ignoreExpireCheckInterval() {
```

Comment:
Maybe you can add `close` to this class?

Reply:
Perhaps I didn't fully grasp your point. Are you suggesting that we should change the method name?